prow/integ-suite-ocp.sh

```diff
@@ -124,5 +124,9 @@ if [ "${TEST_OUTPUT_FORMAT}" == "junit" ]; then
     base_cmd+=" 2>&1 | tee >(${JUNIT_REPORT} > ${ARTIFACTS_DIR}/junit/junit.xml)"
 fi
 
-# Execute the command.
-eval "$base_cmd"
+# Execute the command and capture the exit status
+(eval "$base_cmd")
```

Comment:
why do you need `eval`?

In shell, the exit code is the result of the last command executed, which makes this change redundant. Why not just invoking the command as `$base_cmd` without eval?

Reply:
Check and testing I was using eval because the quoting give me some issues before. I'll check to made a change to fix this